docs/troubleshooting-restore.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_label: Restore
+title: ''
+---
+
+# Troubleshooting restore
+
+:::warning warning
+Try to not restart `rancher-system-agent` on elemental nodes, otherwise it can stuck in following error
+
+`panic: error while connecting to Kubernetes cluster: the server has asked for the client to provide credentials`
+
+If this problem occurs please go with procedure below.
```

Comment:
```suggestion
When a restore is performed, do not restart the `rancher-system-agent` on elemental nodes as it can stale and end with the following error:

`panic: error while connecting to Kubernetes cluster: the server has asked for the client to provide credentials`

If you face this problem, please follow the procedure below.
```

Reply:
updated